external-docs/docs/guides/errors/inspect-command-errors/unexpected-number-of-records.md

```diff
@@ -0,0 +1,9 @@
+# Error - Expected `x` record(s), but found `y` record(s). 
+
+## When it occurs
+
+The inspect command did not find the expected number of records in the data set.
```

Comment:
Can you include the type of records you were looking for in the error message? At the moment it doesn't give clarity what you found too many/too few of.